Description: Updated scripts shebang
 Updated wsh script to use proper shebang corresponding
 to WCC installation.
Author: Philippe Thierry
Last-Update: 2017-05-24
--- a/src/wsh/scripts/md5.wsh
+++ b/src/wsh/scripts/md5.wsh
@@ -1,4 +1,4 @@
-#!/home/jonathan/solution-exp/unlinking/awareness/self/wcc/src/wsh/wsh
+#!/usr/bin/wsh
 
 -- Computing a MD5 sum using cryptographic functions from foreign binaries (eg: sshd/OpenSSL)
 
--- a/src/wsh/scripts/read.wsh
+++ b/src/wsh/scripts/read.wsh
@@ -1,4 +1,4 @@
-#!/home/jonathan/solution-exp/unlinking/awareness/self/wcc/src/wsh/wsh
+#!/usr/bin/wsh
 
  fname="/etc/passwd"
  printf("\n ** Reading file %s\n", fname)
--- a/src/wsh/scripts/dumpbin.wsh
+++ b/src/wsh/scripts/dumpbin.wsh
@@ -1,3 +1,4 @@
+#!/usr/bin/wsh
 
 f = io.open("/bin/pwd", "rb")
 block = 48
--- a/src/wsh/scripts/main.wsh
+++ b/src/wsh/scripts/main.wsh
@@ -1,3 +1,5 @@
+#!/usr/bin/wsh
+
 local ffi = require("ffi")
 ffi.cdef[[
 unsigned long compressBound(unsigned long sourceLen);
